.stylelintrc.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
- [ ] Please as the last reviewer said When the user clicks (or taps) the close (X) button, the popup should disappear I can't see a close button in your design.
![image](https://user-images.githubusercontent.com/74744787/226135826-34948171-c95d-4c92-bb35-b4a6fe4557bf.png)


- [ ] Please as the last reviewer said The popup should be responsive, it fits mobile and desktop sizes properly the button and the image should be in the same place as in the design with proper sizes for mobile and desktop versions. 
Your design 
![image](https://user-images.githubusercontent.com/74744787/226136038-6312490b-fcb6-4e57-b680-8fc2dca59186.png)
Figma 
![Uploading image.png…]()

Reply:
Hi @MeqdamAlqudah 
I'm about to make these changes but at the same time I thought that these are also optional changes.
Can you please confirm this information because I only have 1 more attempt for code review.
Please, I'm in hurry and this 4th block has been really stressfull!

Reply:
both I and the last reviewer request these changes so they are required.